### ISSUE TITLE
Getting started information

### ISSUE THREAD

**Issue by rob100** [OSVR/OSVR-Leap-Motion#18]
Hi,
are there any getting started information for the leap motion plugin available? I have some troubles getting the plugin to work. Especially an example osvr_server_config.json would be nice.
In my case I built the plugin and copied the _com_osvr_LeapMotion.dll_ in the _osvr-plugins-0_ folder from my osvr-server. Now I am stuck, because I don’t know how to modify my _osvr_server_config.json_ file. I tried to add it to the plugins section:

```
{
    "plugins": [
        "com_osvr_LeapMotion"
    ]
}
```

But this ends with the following messages:

```
[OSVR Server] Failed to load the following plugins:
[OSVR Server]  - com_osvr_LeapMotion    Failed to load the module C:/Users/xyz/Downloads/OSVR-Core-Snapshot-v0.6-1115-g26bc354-build245-vs12-64bit/bin/osvr-plugins-0/com_osvr_LeapMotion
```

I would be really happy if you would add some basic instructions to the readme with an short description how to get the plugin working.

**Comment by phiresky** [comment]
Try adding the dlls from the LeapSDK into the osvr-plugins-0 folder. Made it work for me in windows.

**Comment by Taicanium** [comment]
That doesn't seem to be working for me...I've scratched my head a lot trying to figure this out, but nothing seems to be working.

Excerpt from my server config:

> "plugins": [
>       "com_osvr_LeapMotion"
>   ],
>     "drivers": [{
>       "plugin": "com_osvr_LeapMotion",
>       "driver": "ControllerDevice"
>   },

The result:

> [OSVR Server] Failed to load the following plugins:
> [OSVR Server]  - com_osvr_LeapMotion    Failed to load the module C:/Program Files/OSVR/Runtime/bin/osvr-plugins-0/com_osvr_LeapMotion
> [OSVR Server]
> [OSVR Server]
> [OSVR Server] Instantiating configured drivers...
> [OSVR] Added device: org_osvr_filter_videoimufusion/HeadFusion
> [OSVR] Client context initialized for org.osvr.analysisplugin
> [OSVR] Interface initialized for /com_osvr_Multiserver/OSVRHackerDevKitPrediction0/semantic/hmd
> [OSVR] Interface initialized for /com_osvr_VideoBasedHMDTracker/TrackedCamera0_0/semantic/hmd/front
> [OSVR Server] Successes:
> [OSVR Server]  - com_osvr_VideoBasedHMDTracker/VideoBasedHMDTracker
> [OSVR Server]  - org_osvr_filter_videoimufusion/VideoIMUFusion
> [OSVR Server]
> [OSVR Server] Errors:
> [OSVR Server]  - com_osvr_LeapMotion/ControllerDevice   Could not find plugin named com_osvr_LeapMotion

**Comment by leemichaelRazer** [comment]
Yes, you might check that the leap.dll is in the same directory as the OSVR_server.exe file. The plugin itself should reside in the OSVR-plugins-0 directory. Hope that helps.

**Comment by Taicanium** [comment]
Got it. I had the Leap DLLs, but I was missing the DLLs for OpenCV. Got them copied, and it is now working.
